pr-dhl-woocommerce/includes/front-end/class-pr-dhl-front-end-paket.php

```diff
@@ -795,7 +795,7 @@ public function admin_order_add_postnum_field( $fields ) {
 					'label'        => __( 'Post Number', 'dhl-for-woocommerce' ),
 					'required'     => false,
 					'type'         => 'text',
-					'class'        => array( 'shipping-dhl-postnum' ),
+					'class'        => 'short',
```

Comment:
is there a reason for changing the class from `shipping-dhl-postnum` into `short` ?